python/core/auto_generated/layout/qgslayoutitempage.sip.in

```diff
@@ -69,6 +70,13 @@ size was changed. If ``False`` is returned then ``size`` could not be matched
 to a known page size.
 
 .. seealso:: :py:func:`pageSize`
+%End
+
+    QPageLayout pageLayout() const;
+%Docstring
+Returns the page layout for the page, suitable to pass to QPrinter.setPageLayout
+
+.. versionadded:: 3.22
```

Comment:
Let's merge for 3.20 -- there's zero chance of regressions here, and it'll keep the pr queue smaller